includes/class-blocks.php

```diff
@@ -46,7 +46,7 @@ public static function enqueue_block_editor_assets() {
 				'reader_activation_terms' => Reader_Activation::get_setting( 'terms_text' ),
 				'reader_activation_url'   => Reader_Activation::get_setting( 'terms_url' ),
 				'has_recaptcha'           => Recaptcha::can_use_captcha(),
-				'recaptcha_url'           => admin_url( 'admin.php?page=newspack-connections-wizard' ),
+				'recaptcha_url'           => admin_url( 'admin.php?page=newspack-settings' ),
```

Comment:
I think we also need this change in the [Blocks epic](https://github.com/Automattic/newspack-blocks/blob/epic/ia/includes/class-newspack-blocks.php#L212).

Reply:
Good catch! Here's a PR for it: https://github.com/Automattic/newspack-blocks/pull/2045